repo-issues/lib/data-source.js

```diff
@@ -0,0 +1,97 @@
+const http = require('request-promise-native')
+const moment = require('moment')
+const octokit = require('@octokit/rest')()
+
+const show_detaiils_threshold = 5
+const organisation = 'novoda'
+const languageIconMap = {
+    "html": "html5",
+    "groovy": "gradle",
+    "css": "css3",
+    "objective-c" : "c"
+}
+
+var showIssues = false
+var showPRs = true
```

Comment:
Is this ok? I made this global variables to avoid passing the `configuration` around but perhaps there's a better way to do this

Reply:
It would be safer to pass the configuration around rather than keeping some mutable state but in this case `generateViewState` is our only entry point so it's not too bad. 

it's more _correct_ to use `let` instead of `var` though!